addons/calendar/static/src/js/calendar_view.js

```diff
@@ -52,19 +52,7 @@ var AttendeeCalendarPopover = CalendarPopover.extend({
             self.event.record.attendee_status = selectedStatus;  // FIXEME: Maybe we have to reload view
             self.$('.o-calendar-attendee-status-text').text(self.statusInfo[selectedStatus].text);
             self.$('.o-calendar-attendee-status-icon').removeClass(_.values(self.statusColors).join(' ')).addClass(self.statusInfo[selectedStatus].color);
-
-            // Update grid event style
-            var record_id = self.event.record.id;
-            var record_el = $(".fc-event-container > a[data-event-id='" + record_id + "']" );
-            if(selectedStatus === 'needActions') {
-                record_el.removeClass('o_attendee_status_declined').addClass('o_attendee_status_needsAction');
-            }  else if (selectedStatus === 'accepted') {
-                record_el.removeClass('o_attendee_status_declined o_attendee_status_needsAction');
-            } else if (selectedStatus === 'tentative') {
-                record_el.removeClass('o_attendee_status_declined o_attendee_status_needsAction');
-            } else if (selectedStatus === 'declined') {
-                record_el.removeClass('o_attendee_status_needsAction').addClass('o_attendee_status_declined');
-            }
+            self.trigger_up('render_event');
```

Comment:
These changes need to be included in your calendar commit.

This commit should only contain code modifications to `calendar_renderer.js` as it is a (small) global change that impacts all modules using the calendar view.